Specialization/Frost.lua

```diff
@@ -240,7 +227,7 @@ function DeathKnight:FrostGlowCooldowns()
 			)
 
 		MaxDps:GlowCooldown(FR.FrostwyrmsFury, frostwyrmsFuryCooldownTrigger);
-		MaxDps:GlowCooldown(FR.AbominationLimb, abominationLimbCooldownTrigger);
+		MaxDps:GlowCooldown(COMMON.AbominationLimb, abominationLimbCooldownTrigger);
```

Comment:
If you have the talent and are Necrolord, this is a problem right? Moreover, just remove the covenant stuff :P

Reply:
same as before. Spell id doesn't change but I will remove the covenant stuff.